src/napi/module.cpp

```diff
@@ -45,6 +45,7 @@ using namespace std;
 static uv_thread_t fdbThread;
 
 static bool networkStarted = false;
+static int32_t previousApiVersion = 0;
```

Comment:
I'm wondering if you should add header version here too - it feels cleaner to do so, but I'm not sure its actually necessary.

Reply:
Well, i just replicated same code from typescript. I am not sure either.

Reply:
🤷‍♀️ mmm lets leave it for now...